src/components/PhotoGallery/PhotoGallery.tsx

```diff
@@ -28,12 +30,28 @@ const PhotoGallery = () => {
   useEffect(() => {
     getAlbum();
   }, [getAlbum]);
+
   return (
-    <PicturesContainer>
-      {photosURL.map((photo, index) => (
-        <Image src={photo} key={index} />
-      ))}
-    </PicturesContainer>
+    <>
+      {
+        photosURL.length ===0 ?
+        (
+          <Loading />
```

Comment:
Everything looks good! Per @JanitaMahum's feedback, let's wrap the the sub-text and button behind the loader too, so that everything below the "Gallery" heading transition into view all at once once.